tests/haskell_proto_library/BUILD

```diff
@@ -48,6 +53,16 @@ haskell_library(
     ],
 )
 
+haskell_library(
+    name = "hs-lib-b",
+    srcs = ["Bar.hs"],
```

Comment:
It looks like `Bar.hs` only imports `Proto...Person.hs`.  If you make it import `Address.hs` as well I think the build will fail:
https://github.com/tweag/rules_haskell/blob/110721c4434c842a683a0c6f38d9e59327b04b44/tests/haskell_proto_library/Bar.hs#L3

(I'm having difficulty building the repo on my laptop so haven't managed to check it myself yet.)

Reply:
This is correct, I'll read up and try again